helm/alfresco-content-services/charts/alfresco-sync-service/Chart.yaml

```diff
@@ -1,5 +1,4 @@
 apiVersion: v1
-appVersion: 3.3.3.1
```

Comment:
Is this related to search?

Reply:
Fair point, no it's not, I will revert from this PR.